docs/Manual/04 Fusing.md

```diff
@@ -65,13 +65,13 @@ def _():
 schedule1 = nest1.create_schedule()
 ```
 
-Before fusing, both `schedule0` and `schedule1` have a shape of (16, 16). Next we fuse them:
+Before fusing, both `schedule0` and `schedule1` have a shape (16, 16). Now, let’s fuse them:
 ```python
 # Create a fused schedule
 schedule = acc.fuse(schedule0, schedule1)
 f, i, j = schedule.get_indices()
 ```
-Fusing does not change `schedule0` or `schedule1` but rather creates a new fused schedule named `schedule`, whose shape is (2, 16, 16). The first dimension in `schedule` is the so-called fusing dimension `f`, its slice (0, \*, \*) contains a copy of `schedule0`, and its slice (1, \*, \*) contains a copy of `schedule1`.
+Fusing creates a new fused schedule `schedule` with a shape (2, 16, 16), keeping the shapes of `schedule0` and `schedule1` intact. The first dimension in `schedule` is the so-called fusing dimension `f`. Its slice (0, \*, \*) contains a copy of `schedule0`, and its slice (1, \*, \*) contains a copy of `schedule1`.
```

Comment:
It's somewhat important to stress that fusing does not change the original schedules, so I would add that ...:

"Fusing creates a new fused schedule with a shape .... It does not change `schedule0` and `schedule1`."

(Beyond the shape, the original schedules are entirely unchanged, including transformations)